tests/testsuite/hints.rs

```diff
@@ -32,6 +32,21 @@ fn empty_hints_no_warn() {
 
 #[cargo_test]
 fn unknown_hints_warn() {
+    Package::new("bar", "1.0.0")
+        .file(
+            "Cargo.toml",
+            r#"
+            [package]
+            name = "bar"
+            version = "1.0.0"
+            edition = "2015"
+
+            [hints]
+            this-is-an-unknown-hint = true
+            "#,
+        )
+        .file("src/lib.rs", "")
+        .publish();
```

Comment:
This commit us publishing a `bar` package but it seems to be unused by this test.